pipelines/README.md

```diff
@@ -0,0 +1,15 @@
+# Pipeline Definitions
+
+This directory contains pipeline definition YAML files which are used when processing data with the LSST Science Pipelines.
+
+The pipelines defined here come in three flavors: camera-specific (within named directories), camera-agnostic (top-level, if any), and building-block ingredients (within the [\_ingredients](_ingredients) directory).
+Pipelines within the ingredients directory tend to be imported by other pipelines, and are not intended to be used directly by end-users for science purposes.
```

Comment:
The original description sounds a bit odd, as if you were documenting a natural phenomenon:
```suggestion
Pipelines within the ingredients directory are meant to be imported by other pipelines, and are not intended to be used directly by end-users for science purposes.
```